unit-tests/utilities/time/test-waiting-on.cpp

```diff
@@ -162,5 +162,9 @@ TEST_CASE( "Not invoked flush timeout expected" )
     invoked.wait_until( timeout, [&]() { return invoked; } );
     auto wait_time = sw.get_elapsed();
 
+    INFO( wait_time.count() );
+    INFO( timeout.count() );
```

Comment:
What is the difference between 
`wait_time`  and `wait_time.count()`?

Reply:
@noacoohen ?

Reply:
No difference. Fixed it and pushed a new commit.